packages/fxa-admin-panel/scripts/test-ci.sh

```diff
@@ -2,4 +2,4 @@
 
 yarn workspaces focus fxa-admin-panel
 yarn build
-yarn test
+NODE_ENV=test yarn test
```

Comment:
I did this in a few places. It seemed as though running the script would fail if this wasn't set. We also have some inconsistencies in our code around `NODE_ENV=dev` and `NODE_ENV=development`. It'd be great to consolidate on one convention. Maybe I should file a follow up for this?